src/compiler/tools/LanguageService.ts

```diff
@@ -8,7 +8,14 @@ import { Node } from "../ast/common";
 import { SourceFile } from "../ast/module";
 import { FormatCodeSettings, UserPreferences, RenameOptions } from "./inputs";
 import { Program } from "./Program";
-import { DefinitionInfo, EmitOutput, FileTextChanges, ImplementationLocation, RenameLocation, TextChange } from "./results";
+import { DefinitionInfo, EmitOutput, FileTextChanges, ImplementationLocation, RenameLocation, TextChange, DiagnosticWithLocation } from "./results";
+import { RefactorEditInfo } from "./results/RefactorEditInfo";
+import { CodeFixAction } from "./results/CodeFixAction";
```

Comment:
These two imports aren't included in the barrel file (`./results/index.ts`). Could you add them there and then fix these imports?

That should fix these CI errors:

```ts
> ts-node --transpileOnly scripts/verification/ensureNoDeclarationFileErrors
dist-declarations/ts-simple-ast.d.ts:8897:214 - error TS2304: Cannot find name 'RefactorEditInfo'.
8897     getEditsForRefactor(filePathOrSourceFile: string | SourceFile, formatOptions: FormatCodeSettings, positionOrRange: number | TextRange, refactorName: string, actionName: string, preferences?: UserPreferences): RefactorEditInfo | undefined;
                                                                                                                                                                                                                          ~~~~~~~~~~~~~~~~
dist-declarations/ts-simple-ast.d.ts:8907:202 - error TS2304: Cannot find name 'CodeFixAction'.
8907     getCodeFixesAtPosition(filePathOrSourceFile: string | SourceFile, start: number, end: number, errorCodes: ReadonlyArray<number>, formatOptions?: FormatCodeSettings, preferences?: UserPreferences): CodeFixAction[];
                                                                                                                                                                                                              ~~~~~~~~~~~~~
There were declaration file issues!
```